marquez_client/client_wo.py

```diff
@@ -177,30 +179,29 @@ def create_job_run(self, namespace_name, job_name, run_id,
 
         return response
 
-    def mark_job_run_as_started(self, run_id, action_at=None):
-        return self.__mark_job_run_as(run_id, 'start', action_at)
+    def mark_job_run_as_started(self, run_id, at=None):
+        return self.__mark_job_run_as(run_id, 'start', at)
 
-    def mark_job_run_as_completed(self, run_id, action_at=None):
-        return self.__mark_job_run_as(run_id, 'complete', action_at)
+    def mark_job_run_as_completed(self, run_id, at=None):
+        return self.__mark_job_run_as(run_id, 'complete', at)
 
-    def mark_job_run_as_failed(self, run_id, action_at=None):
-        return self.__mark_job_run_as(run_id, 'fail', action_at)
+    def mark_job_run_as_failed(self, run_id, at=None):
+        return self.__mark_job_run_as(run_id, 'fail', at)
 
-    def mark_job_run_as_aborted(self, run_id, action_at=None):
-        return self.__mark_job_run_as(run_id, 'abort', action_at)
+    def mark_job_run_as_aborted(self, run_id, at=None):
+        return self.__mark_job_run_as(run_id, 'abort', at)
 
-    def __mark_job_run_as(self, run_id, action, action_at=None):
+    def __mark_job_run_as(self, run_id, run_state, at=None):
         Utils.is_valid_uuid(run_id, 'run_id')
 
         return self._backend.post(
-            self._path('/jobs/runs/{0}/{1}?at={2}', run_id, action,
-                       action_at if action_at else Utils.utc_now()),
-            headers=_HEADERS,
-            payload={}
+            path=self._path('/jobs/runs/{0}/{1}?at={2}', run_id, run_state,
```

Comment:
just a caveat that the item {1} in the path here is not actually a _run_state_. example are _start_, _complete_, _fail_ etc which is why this was called _action_ (could also be _transition_) and not _run_state_

Reply:
You're absolutely right. Good catch and will rename to `transition` (which is more appropriate than _action_ IMHO)